app/policies/nil_class_policy.rb

```diff
@@ -0,0 +1,13 @@
+class NilClassPolicy < ApplicationPolicy
```

Comment:
What is this class and how exactly you are going to use it?

Reply:
https://github.com/varvet/pundit#nilclasspolicy

Reply:
And where exactly do you have a null object? Do not create classes you have no use for.
Also, I'd be very careful with null objects - by the wave of the hand they are turned into anti-patterns, which complicate the whole flow.